scripts/commit.sh

```
@@ -228,6 +228,7 @@ function handle_ai_commit_generation {
             echo -e "${YELLOW}Edit the AI generated message:${ENDCOLOR}"
             # Create temp file with AI message
             commitmsg_file=$(mktemp "/tmp/commitmsg.XXXXXX")
+            trap "rm -f '$commitmsg_file'" EXIT INT TERM
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)